### PR TITLE
Replace glog with klog

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

File: cmd/cluster-image-registry-operator/main.go [hunk 1]
```diff
@@ -17,34 +17,39 @@ import (
...
-	glog.Infof("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH)
+	klog.Infof("Cluster Image Registry Operator Version: %s", version.Version)
+	klog.Infof("Go Version: %s", runtime.Version())
+	klog.Infof("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH)
 }
 
 func main() {
-	flag.Parse()
-	flag.Lookup("logtostderr").Value.Set("true")
+	klogFlags := flag.NewFlagSet("klog", flag.ExitOnError)
```
Comment (dmage) [inline cmd/cluster-image-registry-operator/main.go]: Why do you need a new flag set?

Comment (coreydaley) [reply cmd/cluster-image-registry-operator/main.go]: Updated in current push

File: cmd/cluster-image-registry-operator/main.go [hunk 2]
```diff
@@ -17,34 +16,34 @@ import (
...
+	klog.Infof("Cluster Image Registry Operator Version: %s", version.Version)
+	klog.Infof("Go Version: %s", runtime.Version())
+	klog.Infof("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH)
 }
 
 func main() {
-	flag.Parse()
-	flag.Lookup("logtostderr").Value.Set("true")
+	klog.InitFlags(nil)
+	defer klog.Flush()
```
Comment (dmage) [inline cmd/cluster-image-registry-operator/main.go]: It seems the operator ignores flags. You can try it with

`make build && WATCH_NAMESPACE=foo ./tmp/_output/bin/cluster-image-registry-operator -skip_headers`

and the operator should stop use prefixes like `I0712 16:45:17.551673   28281 main.go:20]`

**Comment by adambkaplan** [comment]
@coreydaley k8s.io/component-base/logs has a method that initializes klog correctly. It also vendors in klog, so you don't have to declare klog as a dependency.

See https://github.com/openshift/cluster-openshift-apiserver-operator/blob/master/cmd/cluster-openshift-apiserver-operator/main.go#L26 for an example

**Comment by coreydaley** [comment]
@adambkaplan updated per your suggestion

**Comment by adambkaplan** [comment]
@coreydaley looks like bump(*) changes were added alongside regular code changes. Can this be de-tangled?

**Comment by coreydaley** [comment]
@adambkaplan should be fixed now

**Comment by coreydaley** [comment]
/retest

level=info msg="Creating infrastructure resources..." level=error level=error msg="Error: timeout while waiting for state to become 'active' (last state: 'provisioning', timeout: 10m0s)" level=error level=error msg=" on ../tmp/openshift-install-367826706/vpc/master-elb.tf line 19, in resource \"aws_lb\" \"api_external\":" level=error msg=" 19: resource \"aws_lb\" \"api_external\" {" level=error level=error level=fatal msg="failed to fetch Cluster: failed to generate asset \"Cluster\": failed to create cluster: failed to apply using Terraform"

**Comment by coreydaley** [comment]
@adambkaplan rebased, ptal

**Comment by dmage** [comment]
The operator still ignores flags :(

**Pull request merged**